react/components/EXPERIMENTAL_Charts/ScatterChart/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { FC } from 'react'
+import {
+  ScatterChart as ScatterChartBase, 
+  XAxis, 
+  YAxis, 
+  Tooltip,
+  Scatter,
+  CartesianGrid,
+  ResponsiveContainer,
+  ZAxis
+} from 'recharts'
+import PropTypes from 'prop-types'
+import uuid from 'uuid'
+import { commonDefaultProps } from './constants'
+import { getChartDefaultProps, getRangeOfZAxis } from '../helpers'
+import { colors } from '../commonProps'
+
+const CustomTooltip = (props) => {
+  return props.payload.map(item => 
+    <p key={uuid()}>{`${item.dataKey}: ${item.value}`}</p>
```

Comment:
why don't you use `datakey` as `key`?

Reply:
Because `dataKey` indicates the key of data displayed in the axis. Isn't unique.